### ISSUE TITLE
Annotation progress

### ISSUE THREAD

**Issue by glmdgrielson** [afrmtbl/annotationlib#2]
- [x]  Notes
- [x] Speech bubbles - Done.
- [ ] Titles
- [ ] Labels
- [x] Highlights - Still a few inaccuracies.

These are all the major types and the ones around when YouTube stopped letting them be created.

**Comment by afrmtbl** [comment]
Speech bubbles should now work.

**Comment by afrmtbl** [comment]
Implemented highlight annotations in 665b3d959de94805aa612b6b19215531e77f52bd
Fixes https://github.com/afrmtbl/AnnotationsRestored/issues/33

**Comment by glmdgrielson** [comment]
So I can safely check that off?

**Comment by afrmtbl** [comment]
Yeah. There are still font size problems with it (https://github.com/afrmtbl/annotationlib/issues/11) but it works pretty well.
